testing/snapcast/APKBUILD

```diff
@@ -1,21 +1,21 @@
 # Contributor: Carlo Landmeter <clandmeter@gmail.com>
-# Maintainer:
+# Maintainer: Marvin Steadfast <marvin@xsteadfastx.org>
 pkgname=snapcast
 pkgver=0.11.1
 ### NOTE: when bumping pkgver also update sha's below from gitsubmodules
 _jsonrpcpp_sha=115296b2232ba8776467d74602d392d03409ad7e
 _popl_sha=0dc38f8bc30bec1946f7e3dbc5a2d37ad1edb93e
-pkgrel=0
+pkgrel=1
 pkgdesc="Synchronous multi-room audio player"
 url="https://github.com/badaix/snapcast"
 arch="all"
 license="GPL-3.0"
-depends=""
+depends="$pkgname-client $pkgname-server"
```

Comment:
but what if someone wants to install both with `apk add snapcast` it wouldnt install anything or?